src/app/article/guards/activate-article.guard.ts

```diff
@@ -2,17 +2,16 @@ import { Injectable } from '@angular/core';
 import { CanActivate } from '@angular/router';
 import { AuthService } from 'src/app/public/services';
 
-@Injectable({
-  providedIn: 'root'
-})
+@Injectable()
+
 export class ActivateArticleGuard implements CanActivate {
```

Comment:
use [functional guards ](https://medium.com/ngconf/functional-route-guards-in-angular-8829f0e4ca5c) instead